jlm/llvm/frontend/ControlFlowRestructuring.cpp

```diff
@@ -497,6 +497,7 @@ RestructureControlFlow(llvm::cfg * cfg)
   for (const auto & l : tcloops)
     reinsert_tcloop(l);
 
+  // TODO: SLOW
```

Comment:
Noise (and yes, you are right, it is very slow)

Reply:
Oups, yes that was not intentional